src/include/txn_inline.h

```diff
@@ -972,6 +975,23 @@ __wt_txn_read_upd_list_internal(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt,
             continue;
 
         WT_ORDERED_READ(prepare_state, upd->prepare_state);
+
+        /*
```

Comment:
It would be nice if this code were somehow adjacent to txn_resolve_prepared_update_chain that was introduced in WT-6778. That isn't obvious since they exist in different source files (and I guess this function should remain inline?)
If that isn't possible, could the comment here reference that the commit code describes why it needs to resolve in that order?

Reply:
In theory we could un inline it and move it outside to txn.c but that would also take it away from `__wt_txn_read` and `__wt_txn_read_upd_list` which seems like its moving it away from contextual functions. I'll add more detail to the comment.